src/element/mod.rs

```diff
@@ -64,8 +69,8 @@ pub enum Value {
     String(Str),
     Symbol(Symbol),
     Bool(bool),
-    Blob(Vec<u8>),
-    Clob(Vec<u8>),
+    Blob(Bytes),
+    Clob(Bytes),
```

Comment:
🗺️ Notice that we can still handle blob and clob values in a single `match`/`if let` arm:
```rust
if let Blob(b) | Clob(b) = value {
  // ...do stuff with `b`...
}
```